clang/include/clang/APINotes/Types.h

```diff
@@ -600,6 +603,13 @@ class FunctionInfo : public CommonEntityInfo {
 
   friend bool operator==(const FunctionInfo &, const FunctionInfo &);
 
+  FunctionInfo &operator|=(const FunctionInfo &RHS) {
```

Comment:
This function needs to invoke the overload from its superclass as well, similarly to the existing `ObjCPropertyInfo::operator|=`